virttest/virt_vm.py

```diff
@@ -502,7 +502,7 @@ class BaseVM(object):
     LOGIN_TIMEOUT = 10
     LOGIN_WAIT_TIMEOUT = 240
     COPY_FILES_TIMEOUT = 600
-    MIGRATE_TIMEOUT = 3600
+    MIGRATE_TIMEOUT = 2000
```

Comment:
`MIGRATE_TIMEOUT` is used by API like def migrate() uses it for migration timeout.
so let us have `MIGRATE_DOWNTIME = 2000` and set it for `monitor.migrate_set_downtime(self.MIGRATE_DOWNTIME)`